browser/js/common/factories/GridFactory.js

```diff
@@ -285,7 +285,7 @@ app.factory('GridFactory', function($http, $compile, PageFactory, ProjectFactory
         $('#lasso-button-box-' + parentId).remove();
 
         thisWidget.append($compile(" <div class='row nested-buttons'>\
-  <div class='lasso-button-box'>\
+  <div class='lasso-button-box lasso-button-box-hide'>\
```

Comment:
<div class='lasso-button-box lasso-button-box-hide'>\

Can you update to <div class='lasso-button-box lasso-button-box-hide' id='lasso-button-box-"+node.parentId+"''>